django_project/frontend/src/pages/Admin/ContextLayer/StyleConfig/index.jsx

```diff
@@ -21,6 +21,48 @@ import { getLayer } from '../../../Dashboard/LeftPanel/ContextLayers/Layer'
 
 import './style.scss';
 
+const defaultVectorTyleStyle = [
```

Comment:
Thank you so much for this changes
Could you move it to new file?

Reply:
done